pkg/driver/provisioner.go

```
@@ -137,10 +140,12 @@ func (s *ProvisionerServer) DriverRevokeBucketAccess(ctx context.Context,
 
 	klog.InfoS("Deleting user", "id", req.GetAccountId())
 
-	err := s.ntnxIamClient.RemoveUser(ctx, req.GetAccountId())
+	err := s.NtnxIamClient.RemoveUser(ctx, req.GetAccountId())
```

Does ignoring the error mean this is only a best-effort API to revoke user access from bucket, since it returns nil error even if there was an IAM issue for example? Is the caller fine with this behaviour?